### PR TITLE
rename global to vr

### PR DESCRIPTION
**Pull request by drowhunter**
changed vr global name from openVR to vr

### CODE REVIEW

**Comment by drowhunter** [comment]
line 438  of vr_companion.py  not sure where this function is 

`FreePIE_version_2_0_required()`

but maybe it should be renamed to something like 

`Validate_Freepie_Version()`

**Comment by Ofisare** [comment]
> line 438 of vr_companion.py not sure where this function is
> 
> `FreePIE_version_2_0_required()`
> 
> but maybe it should be renamed to something like
> 
> `Validate_Freepie_Version()`

It does not exists, thus it throws an error and stops the script with a "more readable" message :D

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Profiles are found in scripts/profiles and a default profile (WASD_Default.py).
 Custom profiles should be created in scripts/user_profiles which also contains a README.md for **How to define a profile**.
 
 ## Installation
- - download the [complete build with script and profiles](https://github.com/Ofisare/VRCompanion/releases/tag/Release_2.0)
+ - download the [complete build with script and profiles](https://github.com/Ofisare/VRCompanion/releases/latest)
```
Comment (Ofisare) [inline README.md]: Latest is not good, latest might only contain the scripts. This is always a reference to the "latest" complete build with freepie.

File: scripts/vr_companion.py [hunk 2]
```diff
@@ -426,7 +426,7 @@ def selectProfile():
               
 # initialization of state and constants
 if starting:
-    requiredVersion = "2.0"
+    requiredVersion = "3.0"
```
Comment (Ofisare) [inline scripts/vr_companion.py]: Here, I would then also remain at 2.0

File: scripts/README.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ After the update, you have to restart FreePIE, so new functionality can be loade
 If a new version of FreePIE is required the script will communicate this with an error message.
 In that case, download the latest [complete build with scripts and profiles]https://github.com/Ofisare/VRCompanion/releases/tag/Release_2.0).
 
-Under `Settings > Plugins > Open VR` you can switch the vr engine to be used:
+Under `Settings > Plugins > Open VR` you can switch the vr Runtime to be used:
```
Comment (Ofisare) [inline scripts/README.md]: here we missed one open

**Review by Ofisare — CHANGES_REQUESTED** [review]

**Review by Ofisare — APPROVED** [review]

**Pull request merged**